diff --git a/springmvc1/src/main/java/cap/controller/HelloController.java b/springmvc1/src/main/java/cap/controller/HelloController.java
index 51cfca5..27f4a6a 100644
--- a/springmvc1/src/main/java/cap/controller/HelloController.java
+++ b/springmvc1/src/main/java/cap/controller/HelloController.java
@@ -4,8 +4,10 @@ import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.ModelAndView;
 
+import javax.servlet.http.HttpServletRequest;
 import java.util.HashMap;
 import java.util.Map;
 
@@ -55,4 +57,18 @@ public class HelloController {
         map.put("key2", "value2");
         return map;
     }
+
+    @RequestMapping(value = "sayHi", method = RequestMethod.POST)
+    public String hello(HttpServletRequest request, Model model, @RequestParam("name") String username) {
+        String nameByRequest = request.getParameter("name");
+        model.addAttribute("nameByReuest", nameByRequest);
+        model.addAttribute("username", username);
+        return "result";
+
+    }
+
+    @RequestMapping(value = "/sayHi", method = RequestMethod.GET)
+    public String hello() {
+        return "sayHi";
+    }
 }
diff --git a/springmvc1/src/main/webapp/WEB-INF/jsp/result.jsp b/springmvc1/src/main/webapp/WEB-INF/jsp/result.jsp
new file mode 100644
index 0000000..58eecd4
--- /dev/null
+++ b/springmvc1/src/main/webapp/WEB-INF/jsp/result.jsp
@@ -0,0 +1,18 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: liubailin
+  Date: 2018/9/4
+  Time: 11:00
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+${nameByReuest}
+<br>
+${username}
+</body>
+</html>
diff --git a/springmvc1/src/main/webapp/WEB-INF/jsp/sayHi.jsp b/springmvc1/src/main/webapp/WEB-INF/jsp/sayHi.jsp
new file mode 100644
index 0000000..f9ac6c7
--- /dev/null
+++ b/springmvc1/src/main/webapp/WEB-INF/jsp/sayHi.jsp
@@ -0,0 +1,19 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: liubailin
+  Date: 2018/9/4
+  Time: 10:59
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<form action="sayHi" method="post">
+    <input type="text" name="name">
+    <input type="submit" value="submit">
+</form>
+</body>
+</html>
